app/routes/events/components/EventAttendeeStatistics.tsx

```diff
@@ -265,7 +262,7 @@ const EventAttendeeStatistics = ({
       {registrations.length === 0 ? (
         <p style={{ textAlign: 'center' }}>Ingen er påmeldt enda.</p>
       ) : (
-        <>
+        <div style={{ overflow: 'hidden' }}>
```

Comment:
Not a fan of all this inline styling.

Reply:
Sure I don't mind. Moved it to a css file. 👍🏻